core/modules/user/index.ts

```diff
@@ -49,7 +49,7 @@ export const UserModule: StorefrontModule = async function ({store}) {
     }
 
     if (
-      type.endsWith(types.USER_TOKEN_CHANGED)
+      type.endsWith(types.USER_TOKEN_CHANGED) && state.user.token
```

Comment:
are you sure about this second condition? I mean what about the log out/ clearing out the token on purpose? @andrzejewsky

Reply:
I've checked it again - this condition is not even necessary